docs/OWNERS

```diff
@@ -5,7 +5,8 @@ reviewers:
   - smarterclayton
   - thockin
 approvers:
+  - sig-architecture-approvers
```

Comment:
this one seems more like docs approvers?

Reply:
yep.